### ISSUE TITLE
colorize output in web ui

### ISSUE THREAD

**Issue by mark-pictor-csec** [josephburnett/jd#74]
Related to #3 . It'd be nice for the diff output in the ui to be colorized.

I can think of a couple ways of doing this:

* one would be to have the wasm version of the tool produce html output
* another would be to postprocess the output with javascript, turning it into html

Either way, css would be needed for coloring, as well as a radio button to turn on/off.

While you're fiddling with radio buttons and css, it'd also be nice to have a dark theme 😉

**Comment by josephburnett** [comment]
@mark-pictor-csec yes, color in the UI would be great. The UI is just some Golang code that compiles to WASM. The main binds a bunch of callbacks to an existing HTML framework. I could do the HTML rendering in that code too, and include inline colors in each element.

Next time I work on the UI I'll see what I can do. And for the dark theme too ;) Honestly rebuilding the UI from scratch might be a better approach since what we have now is pretty rough around the edges.
